.goreleaser.yaml

```diff
@@ -16,6 +16,8 @@ builds:
   - main: ./cmd/celestia-appd
     binary: celestia-appd
     env:
+      # Enable CGO for Ledger support
+      - CGO_ENABLED=1
```

Comment:
```suggestion
```
I guess some other PR got confused into this

Reply:
Indeed, I merged a wrong branch, though I fixed it!